### PR TITLE
add onlyoffice to drone

### PR DESCRIPTION
**Pull request by voroyam**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -30,6 +30,12 @@ pipeline:
     source: https://github.com/owncloud/user_ldap/releases/download/${LDAP_VERSION}/user_ldap.tar.gz
     sha256: ${LDAP_CHECKSUM}
 
+  onlyoffice:
+    image: plugins/donwload:latest
```
Comment (tboerger) [inline .drone.yml]: A typo

```diff
-    image: plugins/donwload:latest
+    image: plugins/download:latest
```

File: .drone.yml [hunk 2]
```diff
@@ -108,6 +114,8 @@ matrix:
       RICHDOCUMENTS_CHECKSUM: dbca97f24abecb10a23686361e56a1c91c24d9007a5c142169c75a836b8cebed
       LDAP_VERSION: v0.11.0
       LDAP_CHECKSUM: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+      ONLYOFFICE_VERSION: v1.3.0
```
Comment (tboerger) [inline .drone.yml]: Let's add this also for the 10.0.8 matrix permutation, then we don't need to fight with conditions.

Comment (voroyam) [reply .drone.yml]: done

**Comment by tboerger** [comment]
When drone thinks it's green we can merge it and update the other PR.

**Pull request merged**